app/controllers/api/v1/users_controller.rb

```diff
@@ -7,8 +7,12 @@ class UsersController < Api::V1::ApiController
       # before_action :validate_email_update, only: [:show, :destroy, :update]
 
       def index
-        @users = User.all
-        render json: @users, status: :ok
+        result = Users::Leaderboard.call(params: params)
+        if result.success?
+          render(json: { object: result.object, status: :ok })
```

Comment:
The status param must go outside the hash you pass as json.
So it would look like:
```
render(json: { object: result.object }, status: :ok) 
```